src/dattobd.c

```diff
@@ -737,8 +738,16 @@ static inline void dattobd_bio_copy_dev(struct bio *dst, struct bio *src){
 #define BIO_SET_SIZE 256
 #define bio_last_sector(bio) (bio_sector(bio) + (bio_size(bio) / SECTOR_SIZE))
 
-// As of Linux 5.2, __REQ_NR_BITS == 26
-#define __DATTOBD_PASSTHROUGH 30    /* don't perform COW operation */
+/* don't perform COW operation */
+#ifdef HAVE_ENUM_REQ_OP
+// special case for deb9's 4.9 train
+//#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,8,0) && LINUX_VERSION_CODE < KERNEL_VERSION(4,10,0)
```

Comment:
Could we get rid of the commented out code and replace it with an explanation if need be?

Reply:
I'll add more explanation, but I will not remove the version clarification of the feature test macros, since that convention helped me figure out what the issue was faster.